tests/unit/type/stream-cgroups.tcl

```diff
@@ -438,22 +438,22 @@ start_server {
         set reply [r XAUTOCLAIM mystream mygroup consumer2 10 - COUNT 2]
         assert_equal [llength $reply] 2
         set cursor [lindex $reply 0]
-        assert_equal $cursor $id2
+        assert_equal $cursor $id3
         assert_equal [llength [lindex $reply 1]] 2
         assert_equal [llength [lindex $reply 1 0 1]] 2
         assert_equal [lindex $reply 1 0 1] {a 1}
 
         # Claim 2 more entries
-        set reply [r XAUTOCLAIM mystream mygroup consumer2 10 ($cursor COUNT 2]
+        set reply [r XAUTOCLAIM mystream mygroup consumer2 10 $cursor COUNT 2]
         assert_equal [llength $reply] 2
         set cursor [lindex $reply 0]
-        assert_equal $cursor $id4
+        assert_equal $cursor $id5
         assert_equal [llength [lindex $reply 1]] 2
         assert_equal [llength [lindex $reply 1 0 1]] 2
         assert_equal [lindex $reply 1 0 1] {c 3}
 
         # Claim last entry
-        set reply [r XAUTOCLAIM mystream mygroup consumer2 10 ($cursor COUNT 2]
+        set reply [r XAUTOCLAIM mystream mygroup consumer2 10 $cursor COUNT 1]
```

Comment:
i think the `COUNT 2` here was on purpose (asking more than we're gonna get)
any reason you changed that?
```suggestion
        set reply [r XAUTOCLAIM mystream mygroup consumer2 10 $cursor COUNT 2]
```

Reply:
I just changed it to verify we return "0-0" in such a case

Reply:
> you mean in the case we asked for exactly the number of elements that remain?
you think there's a chance this case would have a bug that doesn't also happen when you ask for more?

yes

> anyway, you're saying that the other test (XAUTOCLAIM as an iterator) has coverage for this (running out of entries and asking for more than there are), right?

yes - `XAUTOCLAIM can claim PEL items from another consumer` covers this